ReducingCrime.Rmd

```diff
@@ -172,23 +188,22 @@ corrplot(cor(crimeData[ , (names(crimeData) %in%
                                , "uraban", "pctmin80", "wcon", "wtuc", "wtrd"
                                , "wfir", "wser", "wmfg", "wfed", "wsta", "wloc"
                                , "mix", "pctymle"))])
-         , method="circle", type="upper"
-         , tl.col="Black", bg="Black")
+         , method="circle", type="upper")
```

Comment:
yes background black was bit shady. I feel embarrassed about it. Good that you removed.
But text colour black looked good.